pkg/sql/logictest/logic.go

```diff
@@ -1678,9 +1707,11 @@ func (t *logicTest) setUser(user string, nodeIdxOverride int) func() {
 	}
 	// The default setting for index_recommendations_enabled is true. We do not
 	// want to display index recommendations in logic tests, so we disable them
-	// here.
-	if _, err := db.Exec("SET index_recommendations_enabled = false"); err != nil {
-		t.Fatal(err)
+	// here. This is only applicable if the bootstrap version is >= 22.1.
```

Comment:
It would be nice to parameterize the bootstrap version; then, this would be no different from a version gate.